Two failing tests:

```
[    4s] ======================================================================
[    4s] FAIL: testConfigOrEnvValueErrors (tests.test_configargparse.TestBasicUseCases)
[    4s] ----------------------------------------------------------------------
[    4s] Traceback (most recent call last):
[    4s]   File "/home/abuild/rpmbuild/BUILD/ConfigArgParse-0.14.0/tests/test_configargparse.py", line 526, in testConfigOrEnvValueErrors
[    4s]     env_vars={"VERBOSE" : "bla"})
[    4s]   File "/home/abuild/rpmbuild/BUILD/ConfigArgParse-0.14.0/tests/test_configargparse.py", line 67, in assertParseArgsRaises
[    4s]     *args, **kwargs)
[    4s] AssertionError: "Unexpected value for VERBOSE: 'bla'. Expecting 'true', 'false', 'yes', 'no', '1' or '0'" does not match "unrecognized arguments: discover"
[    4s] 
[    4s] ======================================================================
[    4s] FAIL: test_help_with_metavar (tests.test_configargparse.TestWrappingMetavar)
[    4s] ----------------------------------------------------------------------
[    4s] Traceback (most recent call last):
[    4s]   File "<string>", line 4839, in test_help_with_metavar
[    4s]   File "<string>", line 30, in assertEqual
[    4s] AssertionError: 'usage: python -m unittest [-h] [--proxy <http[s]://example:1234>]\n\noptional arguments:\n  -h, --help            show this help message and exit\n  --proxy <http[s]://example:1234>\n' != 'usage: this_is_spammy_prog_with_a_long_name_sorry_about_the_name\n       [-h] [--proxy <http[s]://example:1234>]\n\noptional arguments:\n  -h, --help            show this help message and exit\n  --proxy <http[s]://example:1234>\n'
[    4s] 
[    4s] ----------------------------------------------------------------------
[    4s] Ran 1657 tests in 1.321s
[    4s] 
[    4s] FAILED (failures=2)
[    4s] 
[    4s] 'usage: python -m unittest [-h] [--proxy <http[s]://example:1234>]\n\noptional arguments:\n  -h, --help            show this help message and exit\n  --proxy <http[s]://example:1234>\n'
[    4s] 'usage: this_is_spammy_prog_with_a_long_name_sorry_about_the_name\n       [-h] [--proxy <http[s]://example:1234>]\n\noptional arguments:\n  -h, --help            show this help message and exit\n  --proxy <http[s]://example:1234>\n'
[    4s] usage: python -m unittest [-h] [--proxy <http[s]://example:1234>]
[    4s] 
[    4s] optional arguments:
[    4s]   -h, --help            show this help message and exit
[    4s]   --proxy <http[s]://example:1234>
[    4s] 
[    4s] usage: this_is_spammy_prog_with_a_long_name_sorry_about_the_name
[    4s]        [-h] [--proxy <http[s]://example:1234>]
[    4s] 
[    4s] optional arguments:
[    4s]   -h, --help            show this help message and exit
[    4s]   --proxy <http[s]://example:1234>
[    4s] 
```
Running version 0.14.0 on openSUSE/Tumbleweed (actually, building package for it). What's going on?
[Full build log](https://github.com/bw2/ConfigArgParse/files/3189121/log.txt)

Seeing this packaging 0.14.0 on Adélie 1.0-beta3 as well.  Did not happen when we were using Python 3.6.5, but is now happening that we bumped to 3.6.8.

```
test_help_with_metavar (tests.test_configargparse.TestWrappingMetavar) ... FAIL

======================================================================
FAIL: test_help_with_metavar (tests.test_configargparse.TestWrappingMetavar)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "<string>", line 5028, in test_help_with_metavar
AssertionError: 'usage: setup.py [-h] [--proxy <http[s]://example[121 chars]4>\n' != 'usage: this_is_spammy_prog_with_a_long_name_sorr[178 chars]4>\n'
+ usage: this_is_spammy_prog_with_a_long_name_sorry_about_the_name
- usage: setup.py [-h] [--proxy <http[s]://example:1234>]
? ^^^^^^^^^^^^^^^
+        [-h] [--proxy <http[s]://example:1234>]
? ^^^^^^
  
  optional arguments:
    -h, --help            show this help message and exit
    --proxy <http[s]://example:1234>


----------------------------------------------------------------------
Ran 1594 tests in 5.663s

FAILED (failures=1)
Test failed: <unittest.runner.TextTestResult run=1594 errors=0 failures=1>
error: Test failed: <unittest.runner.TextTestResult run=1594 errors=0 failures=1>
>>> ERROR: py3-configargparse: check failed
```

--- a/tests/test_configargparse.py
+++ b/tests/test_configargparse.py
@@ -3,6 +3,7 @@ import configargparse
 from contextlib import contextmanager
 import inspect
 import logging
+import os
 import sys
 import tempfile
 import types
@@ -17,6 +18,9 @@ if sys.version_info >= (3, 0):
     from io import StringIO
 else:
     from StringIO import StringIO
+
+# set COLUMNS to get expected wrapping
+os.environ['COLUMNS'] = '80'
 
 # enable logging to simplify debugging
 logger = logging.getLogger()

